### ISSUE TITLE
There's an issue while .ENV value contain '=' char in it

### ISSUE THREAD

**Issue by dzakybrori** [Tienisto/enven#1]
Generation code failed when variable value contain '=' char.  I was trying to generate this .ENV

```
#enven:output=lib/gen/enven.g.dart

#enven:obfuscate=true
#enven:type=String
API_KEY=gAAAAABmgm6E+8JniapOwqbjPfitLkoJh4mK8Q5Q1lVOKEhpwQcYFueR/rMu/7yDsGq/uNfnITr+YjgaSEmv8Nctb9pTKyZzAA==
```

**Comment by Tienisto** [comment]
Fixed in `1.0.1`

**Comment by dhirajReliance** [comment]
> Generation code failed when variable value contain '=' char. I was trying to generate this .ENV
> 
> ```
> #enven:output=lib/gen/enven.g.dart
> 
> #enven:obfuscate=true
> #enven:type=String
> API_KEY=gAAAAABmgm6E+8JniapOwqbjPfitLkoJh4mK8Q5Q1lVOKEhpwQcYFueR/rMu/7yDsGq/uNfnITr+YjgaSEmv8Nctb9pTKyZzAA==
> ```

Hey @dzakybrori  how are you protecting this api in static code analysis during CI/CD. My fortify scanning is integrated in CI/CD process.
